Keda.Cosmosdb.Scaler/src/Repository/ChangeFeedProcessorBuilderFactory.cs

```diff
@@ -0,0 +1,71 @@
+﻿using Keda.CosmosDB.Scaler.Services;
+using Microsoft.Azure.Documents.ChangeFeedProcessor;
+using Microsoft.Azure.Documents.ChangeFeedProcessor.DataAccess;
+using Microsoft.Azure.Documents.Client;

```

Comment:
Not sure what happened to review comments I added in earlier iterations - adding again - move to https://github.com/Azure/azure-cosmos-dotnet-v3

Reply:
The link states that the cosmos dotnet sdk only supports SQL API. Do you not want to support Graph API?

Reply:
That's a good question. https://www.nuget.org/packages/Microsoft.Azure.Cosmos is the latest version and is currently active. Let's ask @brettsam offline what is the recommend approach here.

Reply:
Change Feed Processor only supports SQL API currently, and so do the Functions bindings/trigger

Reply:
Thanks.  So did I misunderstand what our docs say? https://docs.microsoft.com/en-us/azure/azure-functions/functions-integrate-store-unstructured-data-cosmosdb?tabs=csharp

Reply:
@ealsur - Please take a look. I have moved this code to use the V3 SDK package.